src/views/LoginSuccess.vue

```diff
@@ -35,6 +35,8 @@ export default class LoginSuccess extends Vue {
     @Static private request!: ParsedBasicRequest;
     @State private keyguardResult!: KeyguardClient.KeyResult;
 
+    @Action('addWalletAndSetActive') private $addWalletAndSetActive!: (walletInfo: WalletInfo) => any;
```

Comment:
Or `=> void`?

Reply:
Generally yes, but all other occurances of the 'addWallet' and 'setActiveAccount' actions are also `=> any`, so I kept with tradition.